src/crystal/system/windows.cr

```diff
@@ -9,4 +9,8 @@ module Crystal::System
     yield buffer, false
     raise "BUG: retry_wstr_buffer returned"
   end
+
+  def self.to_wstr(path : String, name : String? = nil) : LibC::LPWSTR
+    path.check_no_null_byte(name).to_utf16.to_unsafe
```

Comment:
Since it's not just for paths, the parameter should be named accordingly:
```suggestion
  def self.to_wstr(string : String, name : String? = nil) : LibC::LPWSTR
    string.check_no_null_byte(name).to_utf16.to_unsafe
```